api/all.yaml

```diff
@@ -1408,6 +1428,43 @@ components:
         - data_source_format
         - columns
         - storage_location
+    StagingTableInfo:
+      type: object
+      properties:
+        name:
+          description: Name of table, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        id:
+          description: Unique identifier for the table.
+          type: string
+        staging_location:
+          description: URI generated for the staging table
+          type: string
+        metastore_id:
```

Comment:
Possible to add a cross-link here for metastore id?

Reply:
added a link to the metastore summary API